libs/api/mocks/src/domains/license-service/factories.ts

```diff
@@ -51,24 +46,22 @@ export const genericLicense = factory<GenericLicense>({
 
 export const metadata = factory<GenericUserLicenseMetadata>({
   expired: () => faker.datatype.boolean(),
-  licenseNumber: () => faker.datatype.number().toString(),
+  licenseNumber: () => faker.number.int().toString(),
   title: () => title(),
 })
 
-const dataFieldType = ['Category', 'Group', 'Table', 'Value']
-
 export const genericLicenseDataField = factory<GenericLicenseDataField>({
   description: faker.lorem.words(),
   hideFromServicePortal: faker.datatype.boolean(),
   label: faker.lorem.word(),
-  name: faker.name.findName(),
-  type: faker.random.arrayElement(dataFieldType) as GenericLicenseDataFieldType,
-  value: faker.random.word(),
+  name: faker.person.fullName(),
+  type: faker.helpers.arrayElement(['Category', 'Group', 'Table', 'Value']),
```

Comment:
https://github.com/island-is/island.is/blob/edf13e2ba7c639bf9bd717eb4fc9e3803f8c6e37/libs/api/domains/license-service/src/lib/licenceService.type.ts#L62-L67

```suggestion
  type: faker.helpers.enumValue(GenericLicenseDataFieldType),
```

Reply:
The suggestion is nice but there was a second line that this change was applicable as well. I also needed to change the inports, thats why I couldn't accept the suggestion as is. Was fixed in 72ebd13548f2e90a43b428055c8e832c2ec05619, tho